tests/microbenchmarks.py

```diff
@@ -264,7 +299,9 @@ def C(self):
             obj.A()
             obj.B()
             obj.C()
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def main():
     test_case = TranspileTestCase()
```

Comment:
At column 17: (F405) 'TranspileTestCase' may be undefined, or defined from star imports: utils